internal/pkg/deploy/cloudformation/testdata/rendered_project_cfn_template.yml

```diff
@@ -0,0 +1,128 @@
+# Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+AWSTemplateFormatVersion: '2010-09-09'
```

Comment:
nit: should this file be in the `testdata` directory under the `stack` pkg?

Reply:
sure